src/test/java/pages/MainPage.java

```diff
@@ -0,0 +1,76 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class MainPage extends BasePage {
+    String nameNewPlaylist = "AutomationGuru";
+    String playlistName = "NuGroove";
+    String newName = "NuAutomationGuruGroove";
+    String topPlaylist = ".playlist:nth-child(3)";
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.


```
  private String playlistName = "NuGroove";
  private String newName = "NuAutomationGuruGroove";
  private String topPlaylist = ".playlist:nth-child(3)";
```